app/components/card/Card.js

```diff
@@ -27,7 +27,7 @@ export default class Card extends Component {
 
   render() {
     const placeholder =
-      '../../images/posterholder.png';
+      'images/posterholder.png';
```

Comment:
Does the same path also work after being packaged? The paths in development and production a little different in some cases.